app/views/projects/shared/_with_academy_urn_table.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if projects.empty? %>
+  <%= govuk_inset_text(text: t("project.table.with_academy_urn.empty")) %>
+<% else %>
+  <table class="govuk-table" name="projects_table" aria-label="Projects table">
```

Comment:
Please don't change this, but we do have the GOVUK view components, which includes tables: https://govuk-components.netlify.app/components/table/#input-erb-building-a-table-row-by-row